src/conversions/chrono.rs

```diff
@@ -243,21 +341,30 @@ impl FromPyObject<'_> for FixedOffset {
     /// Note that the conversion will result in precision lost in microseconds as chrono offset
     /// does not supports microseconds.
     fn extract(ob: &PyAny) -> PyResult<FixedOffset> {
-        let py_tzinfo: &PyTzInfo = ob.downcast()?;
-        // Passing `ob.py().None()` (so Python's None) to the `utcoffset` function will only
+        #[cfg(not(Py_LIMITED_API))]
+        let ob: &PyTzInfo = ob.extract()?;
+        #[cfg(Py_LIMITED_API)]
+        {
+            static TZINFO: GILOnceCell<Py<PyType>> = GILOnceCell::new();
+            let tzinfo = TZINFO
+                .get_or_try_init_type_ref(ob.py(), "datetime", "tzinfo")
+                .expect("failed to load datetime.tzinfo");
+            check_type(ob, tzinfo, "PyTzInfo")?;
+        }
+
+        // Passing `()` (so Python's None) to the `utcoffset` function will only
         // work for timezones defined as fixed offsets in Python.
         // Any other timezone would require a datetime as the parameter, and return
         // None if the datetime is not provided.
         // Trying to convert None to a PyDelta in the next line will then fail.
-        let py_timedelta = py_tzinfo.call_method1("utcoffset", (ob.py().None(),))?;
-        let py_timedelta: &PyDelta = py_timedelta.downcast().map_err(|_| {
-            PyTypeError::new_err(format!("{:?} is not a fixed offset timezone", py_tzinfo))
-        })?;
-        let days = py_timedelta.get_days() as i64;
-        let seconds = py_timedelta.get_seconds() as i64;
-        // Here we won't get microseconds as noted before
-        // let microseconds = py_timedelta.get_microseconds() as i64;
-        let total_seconds = Duration::days(days) + Duration::seconds(seconds);
+        let py_timedelta = ob.call_method1("utcoffset", ((),))?;
```

Comment:
If we are touch this, wouldn't passing `()` instead of `((),)` work as well or does `utcoffset` really check for `None` instead of a missing parameter?

Reply:
Sadly not. `utcoffset` indeed checks for `None`